### PR TITLE
Adding micro-frontend-single-spa example

### PR DESCRIPTION
**Pull request by RezaRahmati**
Adding community example for micro-frontend-single-spa

### CODE REVIEW

File: website/versioned_docs/version-5.x/examples.md [hunk 1]
```diff
@@ -46,5 +46,6 @@ sidebar_label: Resources
   - [single-spa-apollo-cleint-auth-with-apollo-federation-backend](https://github.com/hashaneranda/hotel-app) is a single spa frontend example for hotel app with authentication, styled-components, apollo-client for graphql along with apollo federation microservice backend
 - [single-spa-react-angular](https://github.com/nitinreddy3/react-ng-spa-app) a multi-framework example with React and Angular. This app uses routes to render the React and Angular apps.
 - [vite-ts-single-spa-root-config](https://github.com/long-woo/vite-ts-single-spa-root-config): Create a root config example of single-spa using vite and typescript.
+- [micro-frontend-single-spa](https://github.com/RezaRahmati/micro-frontend-single-spa): is a single-sap frontend example including React 18, Angular 14 and a simple backend using nestjs. Sample has features like authetication, routing in react and angular. Also has two versions of root-config, using ejs and pure html+js.
```
Comment (internettrans) [inline website/versioned_docs/version-5.x/examples.md]:
```suggestion
- [micro-frontend-single-spa](https://github.com/RezaRahmati/micro-frontend-single-spa): is a single-spa frontend example including React 18, Angular 14 and a simple backend using nestjs. Sample has features like authetication, routing in react and angular. Also has two versions of root-config, using ejs and pure html+js.
```

**Review by internettrans — APPROVED** [review]

**Comment by internettrans** [comment]
Please fix merge conflicts

**Comment by RezaRahmati** [comment]
@joeldenning thanks, the conflict is resolved

**Comment by MilanKovacic** [comment]
Hi, it seems like example repository did not receive updates in a long time. Feel free to re-open the pull request if you update it.

**Pull request closed without merging**